src/qt/bitcoingui.cpp

```diff
@@ -351,6 +354,11 @@ void BitcoinGUI::createActions()
     // initially disable the debug window menu item
     openRPCConsoleAction->setEnabled(false);
 
+    showMempoolStatsAction = new QAction(platformStyle->TextColorIcon(":/icons/mempoolstats"), tr("&Mempool Statistics"), this);
```

Comment:
@luke-jr: keep in mind that we have permission from the author to distribute them under MIT: https://github.com/bitcoin/bitcoin/issues/6367#issuecomment-118250990